src/yunohost/app.py

```diff
@@ -140,7 +144,7 @@ def app_fetchlist(url=None, name=None):
             raise MoulinetteError(errno.EBADR,
                                   m18n.n('appslist_retrieve_error', error=str(e)))
 
-        if (applist_request.status_code != 200):
+        if applist_request.status_code != 200:
```

Comment:
Didn't look into changing the return code for now.

I moved to logger.error's, which made me wonder if/how we want to check stdout/stderr in the tests (e.g. does it contains the appropriate error string/key)